discovery-provider/src/tasks/entity_manager/entities/social_features.py

```diff
@@ -250,9 +251,7 @@ def validate_social_feature(params: ManageEntityParameters):
     if params.user_id not in params.existing_records[EntityType.USER]:
         raise IndexingValidationError(f"User {params.user_id} does not exist")
 
-    wallet = params.existing_records[EntityType.USER][params.user_id].wallet
-    if wallet and wallet.lower() != params.signer.lower():
-        raise IndexingValidationError(f"User {params.user_id} does not match signer")
+    validate_signer(params)
```

Comment:
There are a couple of places in this file where we call `logger.*`, can you change these to `params.logger.*`? (isaac added `params.logger` recently so we can track entity manager indexing better, it has more context about the tx that it adds to the log. for example we add an `isApp` value `validate_signer`)